lib/web_ui/lib/src/engine/canvaskit/renderer.dart

```diff
@@ -384,47 +353,76 @@ class CanvasKitRenderer implements Renderer {
     frameTimingsOnBuildFinish();
     frameTimingsOnRasterStart();
 
+    // TODO(harryterkelsen): Create and dispose the Rasterizer when the views
+    // are registered/disposed.
+    final Rasterizer rasterizer =
+        _getRasterizerForView(view as EngineFlutterView);
+
     rasterizer.draw((scene as LayerScene).layerTree);
     frameTimingsOnRasterFinish();
   }
 
+  final Map<EngineFlutterView, Rasterizer> _rasterizers =
+      <EngineFlutterView, Rasterizer>{};
+  Rasterizer _getRasterizerForView(EngineFlutterView view) {
+    return _rasterizers.putIfAbsent(view, () {
+      return Rasterizer(view.dom.sceneHost);
+    });
+  }
+
+  /// Returns the [Rasterizer] that has been created for the given [view].
+  /// Used in tests.
+  Rasterizer debugGetRasterizerForView(EngineFlutterView view) {
+    return _getRasterizerForView(view);
```

Comment:
Should this look in the map directly? Delegating to `_getRasterizerForView` will actually end up creating a new rasterizer.

Reply:
No because the test which uses this needs to get a hold of the `Rasterizer` before the first time `renderScene` is called so it isn't created yet.